apps/idos-data-dashboard/src/routes/dashboard/credentials/components/grants-center.tsx

```diff
@@ -100,8 +104,7 @@ const Shares = ({ credentialId, grants }: { credentialId: string; grants: idOSGr
                     variant="outline"
                     colorScheme="red"
                     isDisabled={Boolean(
-                      grant.lockedUntil &&
-                        BigInt(grant.lockedUntil) < BigInt(Math.floor(Date.now() / 1000)),
+                      grant.lockedUntil && grant.lockedUntil * 1000 >= Date.now(),
```

Comment:
```suggestion
                      grant.lockedUntil && timelockToDate(grant.lockedUntil) >= Date.now(),
```
If reusing the function works here I think it's more readable